packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -731,7 +731,12 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 		return (
 			<>
 				<DefaultLineItemSublabel product={ product } />:{ ' ' }
-				{ translate( '%(price)s per month', { args: { price } } ) }
+				{ getLocaleSlug()?.startsWith( 'en' ) || i18n.hasTranslation( '%(price)s per month' )
```

Comment:
I think this is a regression from before https://github.com/Automattic/wp-calypso/pull/81592 was merged.

Reply:
Yes, this branch got into a weird state -- I've created a new PR in #81699 to clean things up.